src/inmanta/server/config.py

```diff
@@ -161,6 +161,16 @@ def default_db_pool_max_size() -> int:
     "server", "ssl_cert_file", None, "SSL certificate file for the server key. Leave blank to disable SSL", is_str_opt
 )
 
+server_compatibility_file = Option(
+    "server",
+    "compatibility_file",
+    "/usr/share/inmanta/compatibility/compatibility.json",
```

Comment:
This default value is not going to work. We only add the compatibility file to the docker containers and not to the RPM. So this would mean that servers, installed from RPM, will not be able to start after an upgrade because the compatibility file is missing. Also, this would mean that starting the server manually (using the inmanta command) on a development machine would not be possible anymore unless you explicitly set this config option to an empty string or put the compatibility file in the right directory. I don't think that is what we want.